GVFS/GVFS.FunctionalTests/FileSystemRunners/BashRunner.cs

```diff
@@ -28,11 +29,18 @@ public class BashRunner : ShellRunner
             "Function not implemented"
         };
 
-        private static string[] permissionDeniedMessage = new string[]
+        private static string[] permissionDeniedWindowsMessage = new string[]
         {
             "Permission denied"
         };
 
+        // TODO(Mac): Update this message when the kext returns a more accurate
+        // error code
+        private static string[] permissionDeniedMacMessage = new string[]
+        {
+            "Resource temporarily unavailable"

```

Comment:
We haven't really settled on what error codes we're going to return, so this is subject to change. That said, does the specific error message matter? Shouldn't we just check that we tried to delete and the delete failed?